en/guide/overriding-sugar-methods.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Overriding sugar methods
+menu: guide
+lang: en
+---
+<div id="page-doc" markdown="1">
+# Overriding sugar methods
+
+Sugar methods are methods added by Express on Node's HTTP request and response objects, which make up the bulk of Express' documented methods. Since the request and response objects are accessible from an instance of Express, you can override their signature and behavior with your own implementation by assigning a custom function.
+
+Here is a very simple example of overriding the [res.send](/{{ page.lang }}/4x/api.html#res.send) method. This will force all request to the app to be rejected with a 403 response.
+
+```js
+var express = require('express')
+var app = express()
+
+app.response.send = function () {
```

Comment:
I was planning on "conditioning" the reader first :), I am fine with removing it.